cli/command/cli_options.go

```diff
@@ -0,0 +1,91 @@
+package command
+
+import (
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/docker/cli/cli/streams"
+	clitypes "github.com/docker/cli/types"
+	"github.com/docker/docker/pkg/term"
+)
+
+// DockerCliOption applies a modification on a DockerCli.
+type DockerCliOption func(cli *DockerCli) error
+
+// WithStandardStreams sets a cli in, out and err streams with the standard streams.
+func WithStandardStreams() DockerCliOption {
+	return func(cli *DockerCli) error {
+		// Set terminal emulation based on platform as required.
+		stdin, stdout, stderr := term.StdStreams()
+		cli.in = streams.NewInStream(stdin)
```

Comment:
If you accept @vdemeester 's suggestion to make the type `streams.In` you'd want to `s/Stream//g` here too.

Reply:
in order not to break (too much) existing code (and I know, this PR WILL break some code), we can alias all the moved Stream related symbols so that command.NewInStream, command.InStream etc still work

Reply:
(on the + side, it will make for less code changes in this PR as well)

Reply:
TBH I'm inclined towards just biting the bullet and making/propagating the change. The necessary updates are pretty mechanical and we should try and avoid building up compatibility debt (which can never be removed) when we don't really need to.

Up to the maintainers though.

Reply:
If it's a one-time change, I agree with grouping breaking changes together (separate commit though)